proto/storage/cluster.proto

```diff
@@ -94,6 +95,43 @@ message AdmissionControllerConfig {
   bool enforce_on_updates = 5;
 }
 
+message ProcessStatus {
+  bool process_enabled = 1;
```

Comment:
We can just call this `enabled`, here and elsewhere. That is also more consistent with other configuration settings in the product.

Reply:
Done